app/frontend/admin/courses/CourseEditor__Root.res

```diff
@@ -395,7 +395,7 @@ let showCourse = course => {
                 href={"/courses/" ++ Course.id(course)}
                 target="_blank"
                 className="inline-flex items-center underline rounded p-1 text-sm font-semibold cursor-pointer text-gray-800 hover:text-primary-500 focus:outline-none focus:text-primary-500 focus:ring-2 focus:ring-inset focus:ring-focusColor-500">
-                <Icon className="if i-external-link-solid mr-2" />
+                <Icon className="if i-external-link-solid me-2" />
```

Comment:
@pupilfirst/designers After thinking about this for a bit, wouldn't it be better for the icon to define how it should be rendered in an RTL environment? Because some icons might want a 90° turn whereas others might want a 180°, and maybe in different directions. I get the feeling that it should be possible to leave this out of the `<Icon>` invocation, and just handle it using details stored as a part of the icon's data by the component.